packages/kit/src/exports/vite/index.js

```diff
@@ -240,16 +154,15 @@ function kit({ svelte_config }) {
 				path.resolve('node_modules'),
 				path.resolve(vite.searchForWorkspaceRoot(cwd), 'node_modules')
 			]);
+
 			// We can only add directories to the allow list, so we find out
 			// if there's a client hooks file and pass its directory
 			const client_hooks = resolve_entry(svelte_config.kit.files.hooks.client);
-			if (client_hooks) {
-				allow.add(path.dirname(client_hooks));
-			}
+			if (client_hooks) allow.add(path.dirname(client_hooks));
 
 			// dev and preview config can be shared
 			/** @type {import('vite').UserConfig} */
-			const result = {
+			const new_config = {
```

Comment:
`new_config` sounds a bit like it's the complete config. maybe `additional_config` to make it clearer that Vite will combine it with the provided config?